code/src/main/resources/site/i18n/phrases.properties

```diff
@@ -1,6 +1,6 @@
 # Admin translations
 site.active = Active
-site.active-help = Activate to display the inquiry box on your website. When a visitor closes the inquiry box or submits the inquiry form, their browser will remember this and prevent it from displaying again unless the identifier has changed in the meantime.
```

Comment:
Did this get removed by accident during the merge conflict resolve? :)

Reply:
Yea, looks like it... Fixed now :)